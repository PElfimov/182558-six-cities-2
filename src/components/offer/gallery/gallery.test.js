import React from 'react';
import renderer from 'react-test-renderer';
import Gallery from './gallery';

const images = [
  `img/room.jpg`,
  `img/apartment-01.jpg`,
  `img/apartment-02.jpg`];
it(`HotelCard correctly renders after relaunch`, () => {
  const tree = renderer
    .create(
        <Gallery
          images={images}
        />).toJSON();
  expect(tree).toMatchSnapshot();
});
