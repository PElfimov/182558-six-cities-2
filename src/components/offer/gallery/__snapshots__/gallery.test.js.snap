// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HotelCard correctly renders after relaunch 1`] = `
<div
  className="property__gallery"
>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/room.jpg"
    />
  </div>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/apartment-01.jpg"
    />
  </div>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/apartment-02.jpg"
    />
  </div>
</div>
`;
